ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_trigonometric_functions.py

```diff
@@ -332,3 +332,48 @@ def test_numpy_deg2rad(
         dtype=dtype,
         subok=True,
     )
+
+# degrees
+@handle_frontend_test(
+    fn_tree="numpy.degrees",
+    dtypes_values_casting=np_frontend_helpers.dtypes_values_casting_dtype(
+        arr_func=[
+            lambda: helpers.dtype_and_values(
+                available_dtypes=helpers.get_dtypes("float"),
```

Comment:
where relevant (ie, functions should accept integer etc parameters and not only float), please use `numeric` instead

Reply:
replaced with numeric

Reply:
should i add numeric to other functions too??